app/src/main/java/com/elavon/converge/processor/ConvergeClient.java

```diff
@@ -113,14 +114,27 @@ public void onFailure(final Call call, final IOException e) {
 
     private Request getRequest(final ElavonRequest request) {
         try {
-            if(!TextUtils.isEmpty(userId) && !TextUtils.isEmpty(merchantId) && !TextUtils.isEmpty(pin)) {
+            if (!TextUtils.isEmpty(userId) && !TextUtils.isEmpty(merchantId) && !TextUtils.isEmpty(pin)) {
                 // set credentials
                 request.setMerchantId(merchantId);
                 Log.d(TAG, "set request merchantId to : " + merchantId);
                 request.setUserId(userId);
                 Log.d(TAG, "set request pin to: " + pin);
                 request.setPin(pin);
                 request.setVendorId("POYNT000");
+                request.setAppVersion("1.0.0");
```

Comment:
Add a comment explaining this for clarity: 
"NOTE: for the middle '...app_name' value above, use p61b or p5 or p61 or similar device type"
i.e:
<ssl_vendor_app_version>1.0.0</ssl_vendor_app_version>
<ssl_vendor_app_name>P61</ssl_vendor_app_name>
<ssl_vendor_id>POYNT000</ssl_vendor_id>